lib/logstash/test_pipeline.rb

```diff
@@ -1,4 +1,3 @@
-require "logstash/pipeline"
```

Comment:
sorry for the late review but we also have `require "logstash/pipeline"` around in a few specs:
- [logstash-integration-rabbitmq](https://github.com/logstash-plugins/logstash-integration-rabbitmq/blob/v7.1.1/spec/outputs/rabbitmq_spec.rb#L3)
- [logstash-mixin-rabbitmq_connection](https://github.com/logstash-plugins/logstash-mixin-rabbitmq_connection/blob/v6.0.0/spec/plugin_mixins/rabbitmq_connection_spec.rb#L3) only used by output-rebbitmq and input-rabbitmq which have been replaced by the integration-rabbitmq plugin
- [logstash-output-jira](https://github.com/logstash-plugins/logstash-output-jira/blob/v3.0.4/spec/outputs/jira_spec.rb#L1) just a test stub and pretty much not supported

Reply:
was there a conscious decision not to keep `require "logstash/pipeline"` around (as a delegate to `require "logstash/java_pipeline"`) ?

Reply:
Thanks for reviewing the PR. I checked the support matrix and missed integration-rabbitmq, mixin-rabbitmq. Most likely I only checked the input- and output-.
We didn't discuss in detail the ways of removing the dependency. My idea was to remove unrelated legacy completely, but I agree given that there are 70+ plugins to maintain, pointing LogStash::Pipeline to JavaPipeline is a better way to keep every plugin include the unsupported one work